CHANGELOG.md

```diff
@@ -81,12 +81,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#600](https://github.com/nf-core/sarek/pull/600) - Remove `TODO` in awsfulltest
 - [#606](https://github.com/nf-core/sarek/pull/606) - Updated `ASCAT` to version `3.0` as module
 - [#608](https://github.com/nf-core/sarek/pull/608) - Prevent candidate VCFs from getting published in manta
+- [#618](https://github.com/nf-core/sarek/pull/618) - Update `multiqc` module
+- [#618](https://github.com/nf-core/sarek/pull/618) - Update test yml files
 - [#620](https://github.com/nf-core/sarek/pull/620) - `gender` is now `sex` in the samplesheet
 - [#630](https://github.com/nf-core/sarek/pull/630) - Update citations file
 - [#632](https://github.com/nf-core/sarek/pull/632) - Update `snpEff` version to `5.1` and cache up to `105`
 - [#632](https://github.com/nf-core/sarek/pull/632) - Update `VEP` version to `106.1` and cache up to `106`
-- [#618](https://github.com/nf-core/sarek/pull/618) - Update `multiqc` module update test yml files
-- [#618](https://github.com/nf-core/sarek/pull/618) - Update test yml files
```

Comment:
@SusiJo I'm assuming that since you put `Update 'multiqc' module update test yml files` and `update test yml files`, you actually meant only `Update 'multiqc' module` and `update test yml files`